gammapy/modeling/models/tests/test_temporal.py

```diff
@@ -55,7 +55,7 @@ def test_energy_dependent_lightcurve(tmp_path):
 
     t = Time(55555.6157407407, format="mjd")
     val = mod.evaluate(t, energy=[0.3, 2] * u.TeV)
-    assert_allclose(val.data, [[2.389591e-21], [4.399548e-23]], rtol=1e-5)
+    assert_allclose(val.data, [[2.278068e-21], [4.280503e-23]], rtol=1e-5)
```

Comment:
why did this change? because of the interpolation scheme?

Reply:
Yes! Previously the interpolation scheme was `method=linear` and `values_scale=linear`. Now it's `method=linear` and `values_scale=log`. This causes the different values in the test.